docs/administration-guide/gaffer-config/proxy.md

```diff
@@ -0,0 +1,3 @@
+!!! info "Work in Progress"
```

Comment:
What will this page contain? There is already a proxy store one

Reply:
Will remove this page as I believe it was intended to contain information on the proxy store but as you say this already exists.